docs/2.generators/badges.md

```diff
@@ -28,15 +28,43 @@ The `badges` generator generates badges for the latest npm version, npm download
 
 ## Arguments
 
-- `name`: The npm package name. By default tries to infer from `package.json`.
-- `github`: Github repository name. By default tries to infer from `package.json`.
-- `license`: Show license badge (requires `github`).
-- `licenseBranch`: Branch to use for license badge defaults to `main`.
-- `bundlephobia`: Show [Bundlephobia](https://bundlephobia.com/) badge (requires `name`).
-- `codecov`: Enable [Codecov](https://codecov.io) badge (requires `github`).
-- `no-npmDownloads`: Hide npm downloads badge.
-- `no-npmVersion`: Hide npm version badge.
-- `provider`: Can be one of `shields` (for [shields.io](https://shields.io/)) or `badgen` / `badgenClassic` (for [badgen.net](https://badgen.net/)). Default is `badgen`.
+::field-group
```

Comment:
This is so nice! I am only thinking if we could think of a (pure) markdown syntax to introduce for undocs (similar to `> [!NOTE]` that can **enhance** option docs like this.

This can be so valuable because this way we can write a readme or docs that can be rendered and readable in Github but also when rendered with undocs more beautifully.

Reply:
~> https://github.com/unjs/undocs/issues/66